workshops/filters.py

```diff
@@ -119,6 +119,13 @@ class EventFilter(AMYFilterSet):
     ]
     status = EventStateFilter(choices=STATUS_CHOICES)
 
+    tags = django_filters.ModelMultipleChoiceFilter(
```

Comment:
This is disputable, but I left this out by purpose, so that the sidebar doesn't take one million pixels of vertical space.